app/models/conversation.rb

```diff
@@ -241,6 +243,12 @@ def mute_period
     6.hours
   end
 
+  def validate_referer_url
+    return unless additional_attributes['referer']
+
+    errors.add(:referer_url, 'invalid') unless url_valid?(additional_attributes['referer'])
```

Comment:
@muhsin-k  rather than throwing an error let's discard the value instead
ref: #https://github.com/chatwoot/chatwoot/issues/4233#issuecomment-1074846146